tools/base/libboard/board_vc/board_vc.c

```diff
@@ -262,8 +276,8 @@ fpga_result read_pcb_info(fpga_token token, char *pcb_info, size_t len)
 
 // Read PKVL information
 fpga_result read_pkvl_info(fpga_token token,
-			fpga_pkvl_info *pkvl_info,
-			int *fpga_mode)
+				fpga_pkvl_info *pkvl_info,
+				int *fpga_mode)
```

Comment:
I thought we were using tabs but then spaces to align with the parameter above. Have you run this through clang-format?

Reply:
fixed